dbt_subprojects/dex/models/_log_decoded_trades/arbitrum/decoded-in-flight/uniswap-v3/uniswap_v3_forks_base_trades_arbitrum.sql

```diff
@@ -0,0 +1,44 @@
+{{ config(
+
+        schema = 'dex_mass_decoding_arbitrum',
+        alias = 'uniswap_v3_base_trades',
```

Comment:
I will try to fix the schema+alias in the next commit.
So the file name and the model name will be changed to `dex_mass_decoding_arbitrum_uniswap_v3_base_trades`.

Do you think this may be too long for a model name?

Reply:
Or `univ3_decoding_arbitrum.base_trades`?